docs/source/restore/restore-tables.rst

```diff
@@ -34,6 +60,8 @@ Without the repair, the restored content of the tables might not be visible, and
 *ONLY AFTER* repairing restored tables, you should set their `tombstone_gc <https://www.scylladb.com/2022/06/30/preventing-data-resurrection-with-repair-based-tombstone-garbage-collection/>`_ option to desired mode (e.g. previous mode or strongly recommended ``tombstone_gc = {'mode': 'repair'}``).
 Without that, purging tombstones for those tables will remain disabled. This can result in great memory consumption over time, as deleted rows won't ever be removed from database's memory.
 
+The last step is to recreate Materialized Views and Secondary Indexes. To do it, just execute index creation and materialized views creation commands, saved before performing the restore.
```

Comment:
Would be useful to add an example for a drop of MV and of SI (in the step above) and for a create of MV and SI.